docs/wiki/boards/current/FLYSPARKF4/FLYSPARKF4.mdx

```diff
@@ -0,0 +1,148 @@
+---
+sidebar_custom_props:
+  specs:
+    target: FLYSPARKF4
+    mcu: STM32F405RGT6
+    imu: ICM42688-P
+    osd: MAX7456EUI+ (SPI 2)
```

Comment:
```suggestion
    osd: MAX7456EUI+ (SPI)
```

- suggest removing from darkmatter board too.
- suggest a smaller font size here

Reply:
the dynamic nature of this area always looks weird, no matter the width of the browser window :(
![image](https://github.com/user-attachments/assets/0dc9d208-ceb7-416d-a4f5-b33eb4769bfd)

Reply:
Yeah definitely needs more work on the info grid, I'll put this PR into draft in the meantime and see what I can do

Reply:
if the PR information is good, i don't mind approving/merging. can do the grid in another PR.